server/api.js

```diff
@@ -156,6 +157,33 @@ router.post("/abilities", authorization, async (req, res) => {
     res.json("inserted");
   }
 });
+//<-------------------deselect learning obj btns---------------------->
+
+router.put("/deselect", async (req, res)=>{
+  try {
+    const learning_obj_id = Number(req.body.learning_obj_id);
+    const student_id = Number(req.body.student_id);
+    const ability = req.body.ability
+    console.log(learning_obj_id);
+    console.log(student_id);
+    console.log(ability);
+    const queryDeselect = `update achievements set  ability= $1
+                      where learning_obj_id = $2 and student_id =$3`; 
+        await Connection.query(queryDeselect, [ability, learning_obj_id, student_id]);
+        res.json("deselected")
```

Comment:
This seem like a repeat of https://github.com/CodeYourFuture/knowledge-checklist/pull/13/files#diff-a813eb0ded684fcb4363a4e760c34103f7db7001843ad3706d560397212010f8R142-R143, is there a reason you can't just use that endpoint ?

Reply:
@petemcfarlane we have made suggested changes can you please check? Also we removed ```Number()``` from `const ability = req.body.ability` from the ability enpoint in order for us to get the null value from `updateAbility(null)` is this okay or is there a better way?
Thanks.